Library/Formula/supervisor.rb

```diff
@@ -0,0 +1,113 @@
+class Supervisor < Formula
+  desc "Process Control System"
+  homepage "http://supervisord.org/"
+  url "https://github.com/Supervisor/supervisor/archive/3.2.1.tar.gz"
+  sha256 "d9c0b17ec42ac6477e7ef0ad2e1f6f1597f855a4d9606c024ac40eb034d6e9ed"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "meld3" do
+    url "https://pypi.python.org/packages/source/m/meld3/meld3-1.0.2.tar.gz"
+    sha256 "f7b754a0fde7a4429b2ebe49409db240b5699385a572501bb0d5627d299f9558"
+  end
+
+  def install
+    inreplace buildpath/"supervisor/skel/sample.conf" do |s|
+      # A `[unix_http_server]` modification
```

Comment:
Still think you can kill all these comments. Additionally, looks like you could just do a single global replace on `/tmp/supervisord\.`